core/services.py

```diff
@@ -102,9 +102,11 @@ def delete_identity(profile: Profile) -> None:
     """
     profile_id = profile.sso_email_id
 
+    profile_services.delete_from_peoplefinder(profile=profile)
```

Comment:
could we name this consistently, if it's doing the equivalent thing?
```suggestion
    profile_services.delete_peoplefinder_profile(profile=profile)
```

I kind of feel like it'd be nicer to replace all of these with a `profile_services.delete` function so core isn't worrying about all the different profile types

Reply:
This sounds a good idea. Actually the logic of managing these profile deletions and the order in which they have to happen etc feels a little too much for core services to be knowledgeable about. I will move all this logic into a delete function within profile services.

Reply:
Done